tests/spread/integration/libarchive-tools/task.yaml

```diff
@@ -8,6 +8,7 @@ execute: |
   chroot "${rootfs}/" bsdtar --version
   chroot "${rootfs}/" bsdunzip --version
   echo "Hello, World" > "${rootfs}/testfile.txt"
+  test "$(chroot foo bsdcat testfile.txt)" = "$(cat foo/testfile.txt)"
```

Comment:
Since `testfile.txt` contains the text `Hello, World`, do we just want to compare it directly to that?
I.e. `test "$(chroot foo bsdcat testfile.txt)" = "Hello, World"`